README.md

```diff
@@ -99,7 +99,9 @@ In Execute block, the following commands are provided.
     - `Assert <boolean expr>[, message]`
     - `AssertEqual <expected>, <got>[, message]`
     - `AssertNotEqual <unexpected>, <got>[, message]`
-    - `AssertThrows <expr>`
+    - `AssertThrows <command>`
+       This will set `g:vader_exception` (from `v:exception`) and
+       `g:vader_throwpoint` (from `v:throwpoint`).
```

Comment:
A minor nit. Can you shift the description to the right with a bullet point? It currently doesn't look nice on the rendered page.

https://github.com/blueyed/vader.vim/blob/ad0257972d67147d0f8e66f2711af11fe329a398/README.md#execute